crates/gpui/src/platform/windows/events.rs

```diff
@@ -1158,7 +1146,7 @@ fn parse_char_msg_keystroke(wparam: WPARAM) -> Option<Keystroke> {
         };
         Some(Keystroke {
             modifiers,
-            key,
+            key: key,
```

Comment:
Why aren't we using field shorthand here anymore?

```suggestion
            key,
```